definitions/scenarios/upgrade_to_satellite_6_4.rb

```diff
@@ -0,0 +1,80 @@
+module Scenarios::Satellite_6_4
+  class Abstract < ForemanMaintain::Scenario
+    def self.upgrade_metadata(&block)
+      metadata do
+        tags :upgrade_to_satellite_6_4
+        confine do
+          feature(:downstream) && feature(:downstream).current_minor_version == '6.3'
+        end
+        instance_eval(&block)
+      end
+    end
+  end
+
+  class PreUpgradeCheck < Abstract
+    upgrade_metadata do
+      description 'Checks before upgrading to Satellite 6.4'
+      tags :pre_upgrade_checks
+      run_strategy :fail_slow
+    end
+
+    def compose
+      add_steps(find_checks(:default))
+      add_steps(find_checks(:pre_upgrade))
+      add_step(Checks::RemoteExecution::VerifySettingsFileAlreadyExists.new)
```

Comment:
For this check, we have added [confine](https://github.com/theforeman/foreman_maintain/blob/master/definitions/checks/remote_execution/verify_settings_file_already_exists.rb#L8) block which checks for `downstream` feature plus condition `feature(:downstream).current_minor_version == '6.2'`.

If this check required here for sat 6.4 then we will need to modify this condition.

Reply:
Good point, removed. Not needed after it was migrated already